nss-copper-dragonflies/WorkforceManagement
# Ap employee edit

# Description
[Completed ticket 4] #4 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout "AP-EmployeeEdit"

test editing all fields of an employee
try adding more than one training programs and adding none
try changing computer and check if unassigned date works
try adding a computer to a new employee

# Checklist:
